app/views/candidate_mailer/interview_cancelled.text.erb

```diff
@@ -9,5 +9,3 @@ They’ve given the following reason for cancelling the interview:
 ^ <%= @reason %>
 
 Contact <%= @provider_name %> if you have any questions.
-
-<%= render 'get_support' %>
```

Comment:
Shall we delete the get support partial and just put the relevant content directly into the layout? Since we don't need to reuse it anymore